extensions/sns/extension.json

```diff
@@ -54,24 +48,35 @@
     "propose": {
       "about": "Subcommand for submitting a CreateServiceNervousSystem NNS Proposal.",
       "args": {
-        "init_config_file": {
-          "about": "Path to a configuration file specifying the SNS to be created.",
+        "network": {
+          "about": "The network to deploy to. This can be \"local\", \"ic\", or the URL of an IC network.",
```

Comment:
No change required.

Maybe, mention that this does the same thing as the `--network` flag of `dfx canister`?

Reply:
I think improving these messages should be left for a future MR where they can be changed systematically (when changed here, they should also be changed in the actual help message in the sns cli library in the monorepo)

Reply:
I mean that we need to change it in two spots (this file, and in the monorepo's sns cli)